### PR TITLE
PR - 52758 - Increase the space between the columns 

### PR DESCRIPTION
**Pull request by meglenapnovoselska**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/SummaryEvidence.cshtml [hunk 1]
```diff
@@ -46,16 +46,16 @@
                     <caption class="govuk-table__caption govuk-visually-hidden">Obligation evidence summary</caption>
                     <thead class="govuk-table__head">
                         <tr class="govuk-table__row">
-                            <th class="govuk-table__header">
+                            <th class="govuk-table__header govuk-!-text-align-left" style="width: 1%;">
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/SummaryEvidence.cshtml:49]: @meglenapnovoselska is 1% width here correct, think that probably needs to reflect the actual % width of the column

Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/SummaryEvidence.cshtml]: @JohnPadgettCivica agree, it's changed.

File: src/EA.Weee.Web/Views/Shared/_EvidenceReportHintText.cshtml [hunk 2]
```diff
@@ -1 +1,3 @@
-﻿<p class="govuk-details__text">The data shown in the table is for household evidence.<br />The evidence column includes approved evidence and net transfer.</p>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Views/Shared/_EvidenceReportHintText.cshtml:1]: @meglenapnovoselska I think this hint text maty be used elsewhere. May be worth pulling this hint text out so it is specific to the PCS.

Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Views/Shared/_EvidenceReportHintText.cshtml:1]: okay, I will

**Pull request merged**